client/lib/analytics/test/google-analytics-4.js

```diff
@@ -47,6 +46,12 @@ describe( 'Google Analytics 4 implementation', () => {
 				TRACKING_IDS.wpcomGoogleGA4Gtag,
 				params
 			);
+
+			expect( window.gtag ).not.toHaveBeenCalledWith(
+				'config',
+				TRACKING_IDS.jetpackGoogleGA4Gtag,
+				params
+			);
```

Comment:
One small idea, when during testing we care only about a specific param being matched, here are the first two ones, we can ignore the rest so that our assertion will not "succeed falsely"
```suggestion
			expect( window.gtag ).not.toHaveBeenCalledWith(
				'config',
				TRACKING_IDS.jetpackGoogleGA4Gtag,
				expect.anything()
			);
```